slices/grep.yaml

```diff
@@ -0,0 +1,20 @@
+package: grep
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libpcre3_libs
```

Comment:
You will actually need [libpcre2-8-0](https://packages.ubuntu.com/noble/libpcre2-8-0) for noble: https://packages.ubuntu.com/noble/grep.

Reply:
true!